pkg/registry/generic/etcd/etcd_test.go

```diff
@@ -97,13 +97,21 @@ func (sm SetMatcher) Matches(obj runtime.Object) (bool, error) {
 	return sm.Has(pod.Name), nil
 }
 
-// EverythingMatcher matches everything
-type EverythingMatcher struct{}
+func (setMatcher) MatchesSingle() (string, bool) {
+	return "", false
```

Comment:
IIUC setMatcher contains only the list of names that we would like to match (at least it's how I understood it from Matches method and its line 97)